packages/amplify-e2e-core/src/configure/index.ts

```diff
@@ -26,6 +26,11 @@ export const amplifyRegions = [
   'ap-southeast-2',
   'ap-south-1',
   'ca-central-1',
+  'us-west-1',
```

Comment:
nit: move this up in front of `us-west-2` to keep the ordering consistent - for this line and others

Reply:
@jhockett updated